/**:
  ros__parameters:
    service_name: "/spgc/trigger_service"
    default_string: "No service available"